subsys/bluetooth/mesh/light_temp_srv.c

```diff
@@ -189,23 +189,22 @@ static void lvl_delta_set(struct bt_mesh_lvl_srv *lvl_srv,
 		CONTAINER_OF(lvl_srv, struct bt_mesh_light_temp_srv, lvl);
 	struct bt_mesh_light_temp_status status = { 0 };
 	struct bt_mesh_light_temp_cb_set cb_msg;
-	int16_t start_value = temp_to_lvl(srv, srv->temp_last);
-	int16_t target_value;
+	int16_t start, target;
```

Comment:
Everything I said that applies to start also applies to target. Target_lvl maybe?

Reply:
Changed. Notice that in lightness_srv, we're using `target_actual`, as this target value is using the "Lightness Actual" representation.